### DISCUSSION TITLE
[Projects Beta]: Repo filter removes draft issues

### DISCUSSION THREAD

**Discussion by RobinReiner** [community/community#7826] [Projects and Issues]
There are several repositories in my project. When I create repository-specific views and create a draft in them, it is not displayed.

What I have in mind:
-  Use repo as filter for drafts what is already suggested by the filter: `repo:"myRepo",repo`
- Combine Filters: `repo:"my repo", no:repo`

My Workaround to see the drafts I need to create a filter that excludes all repositories except the desired one.
`-repo:"excludedRepo1","excludedRepo2"`

Or does the possibility already exist and I am doing something fundamentally wrong?

Cheers

**Comment by azenMatt** [comment]
This looks like you have spotted a bug, it currently doesn't work to say:

`repo:x, no:repo` 

Which should give you everything in repo x OR everything with no repo. 

We will get this fixed
